delivery_schenker/models/delivery_carrier.py

```diff
@@ -179,6 +179,13 @@ class DeliveryCarrier(models.Model):
         help="If set, this contact will be sent as invoice address to Schenker."
         "\nIf Address ID is set, it will be part of it instead of the sender",
     )
+    is_schenker_submit_booking = fields.Boolean(
+        string="Submit Booking",
+        default=True,
+        help="Submit Booking On Schenker site."
+        "\nIf set, submits booking on site automatically."
+        "\nIf not set, user has to submit booking manually.",
+    )
```

Comment:
Could you please change the name of the field to schenker_submit_booking
```suggestion
    schenker_submit_booking = fields.Boolean(
        string="Submit Booking",
        default=True,
        help="Submit Booking On Schenker site."
        "\nIf set, submits booking on site automatically."
        "\nIf not set, user has to submit booking manually.",
    )
```